provisioning/roles/nginx/README.md

```diff
@@ -0,0 +1,172 @@
+nginx
```

Comment:
This is a checked-in copy of the code in https://github.com/bennojoy/nginx/pull/14. It's only copied here to get access to those changes without requiring the joys of git submodules or a forked playbook over on `ansible-galaxy`.